cmd/gcp-controller-manager/node_annotator.go

```diff
@@ -311,6 +310,44 @@ func extractKubeLabels(instance *compute.Instance) (map[string]string, error) {
 	return parsedLabels, nil
 }
 
+func extractResizeRequestLabel(instance *compute.Instance) *string {
+	const createdByRRKey = "google-compute-mig-resize-request"
+
+	if instance.Metadata == nil {
+		return nil
+	}
+
+	createdByRR, err := findValue(instance.Metadata.Items, createdByRRKey, instance.SelfLink)
+	if err != nil {
+		klog.Warningf("While finding %q key got error: %v", createdByRRKey, err)
+		return nil
+	}
+	if createdByRR == nil {
+		return nil
+	}
+
+	if elements := strings.Split(*createdByRR, "/"); len(elements) > 1 {
```

Comment:
A comment here on what *createByRR format is and what you are trying to extract would be helpful.

Reply:
Added a comment.